### PR TITLE
Ports 4 new signs from Europa and tweaks bar sign to allow big signs

### PR DESCRIPTION
**Pull request by jknpj**
eurosigns
<s>https://webmshare.com/GZ568</s>
https://webmshare.com/aD67j [i fixed the white crap just need to commit it]
7 actually, but two of them are currently not used on/off stages.

:cl:
 * rsadd: Ports 4 new signs from Europa

### CODE REVIEW

**Comment by kol1th** [comment]
64x64 is the future

**Comment by N3X15** [comment]
Get rid of the one with the floating chick above it, that'll block visiblility of the tiles behind it.

**Comment by jknpj** [comment]
It's not that bad i can even see a salt shaker in the grou- OH MY GOD PLAAAAANES

planes
literally my whole planes

**Comment by kol1th** [comment]
Nothing wrong with it tbh.

**Comment by SangriaMonkey** [comment]
It's a little obnoxious.

**Comment by Ephx1** [comment]
It's _very_ obnoxious

**Comment by Probe1** [comment]
Slide the Slyph to a different PR so this one isn't obstructed by discussion imo

**Comment by 9600bauds** [comment]
Yeah, what Probbu said

File: code/game/objects/structures/barsigns/bay.dm [hunk 1]
```diff
@@ -112,3 +112,33 @@
 /datum/barsign/bay/bark
 	name = "The Bark"
 	icon = "bark"
+
+/datum/barsign/bay/topmen
+	name = "Top Men"
+	icon = "topmen"
+	pixel_x = -16
+	pixel_y = -31
```
Comment (Probe1) [inline code/game/objects/structures/barsigns/bay.dm:120]: What are these pixel offsets accomplishing?

Comment (Shadowmech88) [reply code/game/objects/structures/barsigns/bay.dm:120]: Offsetting by negative pixels causes the icon to be resized by those dimensions. So for example, you can make a 48x48 icon occupy 32x32 space in-game by offsetting its `pixel_x` and `pixel_y` by -16.

Comment (jknpj) [reply code/game/objects/structures/barsigns/bay.dm:120]: They make the over 62x32 signs appear in the right place. Our old barsigns are 64x32 so they have both offsets 0 and fit perfectly but the new ones ara bigger so you actually have to place them using a minus offset

Comment (PJB3005) [reply code/game/objects/structures/barsigns/bay.dm:120]: These pixel values needs to be multiplied by `PIXEL_MULTIPLIER`

**Review by PJB3005 — CHANGES_REQUESTED** [review]

File: code/game/objects/structures/barsigns/bay.dm (hunk 1 above)
Comment (jknpj) [reply code/game/objects/structures/barsigns/bay.dm:120]: tossing it in barsign.dm to avoid ctrl+c ctrl+v

**Review by PJB3005 — APPROVED** [review]

**Pull request merged**